src/Microsoft.Azure.SignalR/ServerConnections/ServiceConnection.Log.cs

```diff
@@ -44,8 +44,8 @@ private static class Log
             private static readonly Action<ILogger, string, Exception> _serviceConnectionConnected =
                 LoggerMessage.Define<string>(LogLevel.Debug, new EventId(20, "ServiceConnectionConnected"), "Service connection {ServiceConnectionId} connected.");
 
-            private static readonly Action<ILogger, Exception> _applicationTaskTimedOut =
-                LoggerMessage.Define(LogLevel.Error, new EventId(21, "ApplicationTaskTimedOut"), "Timed out waiting for the application task to complete.");
+            private static readonly Action<ILogger, Exception> _applicationTaskCancelled =
+                LoggerMessage.Define(LogLevel.Error, new EventId(21, "ApplicationTaskCancelled"), "Cancelled running application code, probably caused by time out.");
```

Comment:
nit: Canceled

Reply:
According to https://www.grammarly.com/blog/canceled-vs-cancelled/https://grammar.yourdictionary.com/grammar/spelling-and-word-lists/cancelled-or-canceled-what-s-the-correct-spelling.html, both `canceled` and `cancelled` are correct? 😃

Reply:
Yeah, but American English is preferred over British English when exposing logs and APIs.

Couple examples:
https://source.dot.net/#System.Private.CoreLib/Task.cs,c0742f865ee9be11,references
https://source.dot.net/#System.Private.CoreLib/Task.cs,3dc5e4fb145ac32f
https://source.dot.net/#System.Private.CoreLib/OperationCanceledException.cs,a0a8b79c277cf327

Reply:
Thanks for sharing! will update